sw/device/lib/dif/dif_uart.c

```diff
@@ -539,3 +539,25 @@ dif_uart_result_t dif_uart_loopback_set(const dif_uart_t *uart,
 
   return kDifUartOk;
 }
+
+dif_uart_result_t dif_uart_set_timeout(const dif_uart_t *uart, uint32_t value,
+                                       dif_uart_toggle_t enable) {
```

Comment:
Are you required the pass a value to disable the timeout? Maybe it makes more sense to have separate `enable_timeout` and `disable_timeout` functions, where the former accepts a timeout value.

Alternatively, require that for disable, a specific `value` is used?

Reply:
Added separate `enable` and `disable` functions. The `disable` function as written will reset the timeout value (duration) to 0 ticks. This isn't strictly necessary but seems reasonable. If needed the preexisting timeout value can be saved using `_get_rx_timeout` before disabling the timeout.